connector/src/main/java/org/geysermc/connector/scoreboard/Scoreboard.java

```diff
@@ -42,14 +42,15 @@
 
 import static org.geysermc.connector.scoreboard.UpdateType.*;
 
-@Getter
 public final class Scoreboard {
     private final GeyserSession session;
     private final GeyserLogger logger;
+    @Getter
     private final AtomicLong nextId = new AtomicLong(0);
 
-    private final Map<String, Objective> objectives = new ConcurrentHashMap<>();
-    private final Map<String, Team> teams = new HashMap<>();
+    private final Map<String, Objective> objectives = new HashMap<>();
```

Comment:
This appears to be accessed on multiple threads (both the translators and the onUpdate).